src/components/RoomView/roomView.css

```diff
@@ -74,29 +74,38 @@ body {
   justify-content: center;
   margin-top: 2%;
 }
-.setUpGame {
+.setUpContainer {
```

Comment:
Personally I think having a css file just for this component will help - we get a lot of conflicts in this css file

Reply:
Additionally, it looks like roomview css has a lot of styles applying to this?
<img width="387" alt="Screen Shot 2023-02-10 at 2 23 37 PM" src="https://user-images.githubusercontent.com/71895647/218179705-a71792d0-c9df-495c-b952-e3dcc9d79fd0.png">

Reply:
I have to step out but maybe we need to discuss how we want this to look. i think if it doesnt have a different bg color it's hard to see. tbh i kind of like its placement and color before. something like this..
<img width="898" alt="Screen Shot 2023-02-10 at 2 28 16 PM" src="https://user-images.githubusercontent.com/71895647/218180354-027df390-755c-49be-81fd-3ea4aff07193.png">

Reply:
IGNORE-  my cache was displaying old stuff (facepalm)